{{cookiecutter.project_slug}}/requirements/base.txt

```diff
@@ -24,7 +24,7 @@ flower==0.9.5  # https://github.com/mher/flower
 {%- endif %}
 {%- endif %}
 {%- if cookiecutter.use_async == 'y' %}
-uvicorn==0.12.2  # https://github.com/encode/uvicorn
+uvicorn[standard]==0.12.2  # https://github.com/encode/uvicorn
 wsproto==0.15.0  # https://github.com/python-hyper/wsproto/
```

Comment:
```suggestion
```

Remove wsproto until this issue is fixed: https://github.com/encode/uvicorn/issues/811